index.ipynb

```diff
@@ -47,20 +53,23 @@
     "  This kata introduces a 3-qubit error correcting code for protecting against bit-flip errors.\n",
     "\n",
     "\n",
-    "For a full list of Quantum Katas, available as Q# projects, see the [QuantumKatas repository](https://github.com/Microsoft/QuantumKatas#list-of-katas-).\n",
+    "For a full list of Quantum Katas, available as Q# projects instead of Jupyter Notebooks, see the [QuantumKatas repository](https://github.com/Microsoft/QuantumKatas#list-of-katas-).\n",
```

Comment:
```suggestion
    "For a full list of Quantum Katas available as Q# projects instead of Jupyter Notebooks, see the [QuantumKatas repository](https://github.com/Microsoft/QuantumKatas#list-of-katas-).\n",
```